pesm.so/src/action.cpp

```diff
@@ -229,7 +229,7 @@ int pesm_action::run(void) {
  * @return 0 - success. non-zero otherwise
  *
  * */
-int pesm_action::do_gpu_list() {
+int ddddo_gpu_list() {
```

Comment:
Do we still need this function as its replaced by display_gpu_info() ?

Reply:
done